src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -205,7 +205,7 @@ function test_zap() {
     local osd_data=$dir/dir
     $mkdir -p $osd_data
 
-    ${CEPH_DISK} $CEPH_DISK_ARGS zap $osd_data 2>&1 | grep -q 'not full block device' || return 1
+    ${CEPH_DISK} $CEPH_DISK_ARGS zap $osd_data 2>&1 | grep -q 'not full device' || return 1
```

Comment:
@wjwithagen why did you change the grepped error message?

```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/ceph-disk/tests/ceph-disk.sh:208: test_zap:  grep -q 'not full device'
/home/jenkins-build/build/workspace/ceph-pull-requests/src/ceph-disk/tests/ceph-disk.sh:208: test_zap:  /home/jenkins-build/build/workspace/ceph-pull-requests/src/ceph-disk/.tox/py27/bin/coverage run --append --source=ceph_disk -- /home/jenkins-build/build/workspace/ceph-pull-requests/src/ceph-disk/.tox/py27/bin/ceph-disk --verbose --prepend-to-path= --statedir=td/test-ceph-disk --sysconfdir=td/test-ceph-disk zap td/test-ceph-disk/dir
/home/jenkins-build/build/workspace/ceph-pull-requests/src/ceph-disk/tests/ceph-disk.sh:208: test_zap:  return 1
```

and it looks like "make check" failed due to this change, see https://jenkins.ceph.com/job/ceph-pull-requests/28056/consoleFull#-23068421240526d21-3511-427d-909c-dd086c0d1034.